app/internal/page/choose_attorneys_address_test.go

```diff
@@ -173,9 +173,15 @@ func TestPostChooseAttorneysAddressManual(t *testing.T) {
 		On("Get", mock.Anything, "session-id").
 		Return(&Lpa{Attorneys: []Attorney{attorneyWithoutAddress}}, nil)
 
+	attorneyWithoutAddress.Address.Line1 = "a"
+	attorneyWithoutAddress.Address.Line2 = "b"
+	attorneyWithoutAddress.Address.Line3 = "c"
+	attorneyWithoutAddress.Address.TownOrCity = "d"
+	attorneyWithoutAddress.Address.Postcode = "e"
```

Comment:
kind of confusing that you are adding an address to `attorneyWithoutAddress`, this is why I prefer putting the struct itself in the tests to make it obvious what is being done

Reply:
True - I'll admit I was being lazy here fixing the tests 🙄